docs/source/install/deb.rst

```diff
@@ -2,9 +2,15 @@ Ubuntu / Debian
 ===============
 
 This guide provides step-by step instructions for installing StackStorm on a single Ubuntu/Debian system per
-the :doc:`Reference deployment </install/overview>`. The script `st2bootstrap-deb.sh
-<https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-deb.sh>`_ codifies the
-instructions below.
+the :doc:`Reference deployment </install/overview>`.
+
+.. rubric:: TL;DR
```

Comment:
We should start utilizing `include` directive more often for things like that (check some of my previous PRs of how I do it).

Especially the installer needs some include love, way too much duplicated stuff :P